includes/class-wciep-settings.php

```diff
@@ -589,7 +539,7 @@ public function icons_radio_element_callback( $args ) {
 			$radios .= sprintf( '<td style="padding-top:0" align="center"><input type="radio" class="radio" id="%1$s[%2$s][%3$s]" name="%1$s[%2$s]" value="%3$s"%4$s /></td>', esc_attr( $menu ), esc_attr( $id ), esc_attr( $key ), checked( $current, $key, false ) );
 		}
 		$html = '<table><tr>'.$icons.'</tr><tr>'.$radios.'</tr></table>';
-		$html .= '<p class="description"><i>'. __('<strong>Please note:</strong> you need to open your website in a new tab/browser window after updating the cart icon for the change to be visible!','woocommerce-improved-external-products').'</p>';
+		$html .= '<p class="description"><i>'. wp_kses( __('<strong>Please note:</strong> you need to open your website in a new tab/browser window after updating the cart icon for the change to be visible!','woocommerce-improved-external-products') ) .'</p>';
```

Comment:
I think this one could be improved. Maybe you can use `sprintf` to detach the "Please note:" and in the end you can use `wp_kses_post`.

Reply:
This method has not been used since it was created; I removed it.